distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection.enso

```diff
@@ -218,3 +234,26 @@ type Connection
     drop_table : Text -> Nothing
     drop_table self table_name =
         self.execute_update (self.dialect.generate_sql (Query.Drop_Table table_name))
+
+## ADVANCED
+   The default types we want to include in `tables`.
+default_table_types =
+    ["TABLE", "VIEW", "TEMPORARY TABLE", "TEMPORARY VIEW", "MATERIALIZED VIEW", "FOREIGN TABLE", "PARTITIONED TABLE", "GLOBAL TEMPORARY"]
```

Comment:
this should be dialect based I think (though probably only a minor thing).
Should be PRIVATE too,